docker_test.go

```diff
@@ -361,6 +423,86 @@ func run(m *testing.M) int {
 	webhookurl = webhook.Server.URL
 	fmt.Println("webhookurl", webhookurl)
 
+
+	// Setup MINIO
+	var minioClient *minio.Client
+
+	options := &dockertest.RunOptions{
+		Repository: "minio/minio",
+		Tag:        "latest",
+		Cmd:        []string{"server", "/data"},
+		PortBindings: map[dc.Port][]dc.PortBinding{
+			"9000/tcp": []dc.PortBinding{{HostPort: "9000"}},
+		},
+		Env: []string{"MINIO_ACCESS_KEY=MYACCESSKEY", "MINIO_SECRET_KEY=MYSECRETKEY"},
+	}
+
+	resource, err := pool.RunWithOptions(options)
+	if err != nil {
+		log.Fatalf("Could not start resource: %s", err)
+	}
+
+	endpoint := fmt.Sprintf("localhost:%s", resource.GetPort("9000/tcp"))
```

Comment:
`endpoint` is too generic here

```suggestion
	minioEndpoint := fmt.Sprintf("localhost:%s", resource.GetPort("9000/tcp"))
```